### ISSUE TITLE
Failed to create workspace

### ISSUE THREAD

**Issue by aboucham** [fabric8io/fabric8-platform#699]
When we click on Create workspace , we receive the following error:

![failed_create_workspace](https://user-images.githubusercontent.com/29703868/29208459-115758aa-7e82-11e7-9e20-596647342c2b.png)

**Comment by rawlingsj** [comment]
Yeah this is definitely an issue that we're seeing.  Hopefully will look into in very soon!
